README.md

```diff
@@ -2,6 +2,13 @@
 description: Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 ---
 
+{% hint style="danger" %}
```

Comment:
should we mark this as info?

Reply:
Got it! I had it as Danger purely because I thought that that would make it stand out more.